include/SFML/Audio/Music.hpp

```diff
@@ -140,7 +140,7 @@ class SFML_AUDIO_API Music : public SoundStream
     /// \see openFromFile, openFromStream
     ///
     ////////////////////////////////////////////////////////////
-    [[nodiscard]] bool openFromMemory(const void* data, std::size_t sizeInBytes);
+    [[nodiscard]] bool openFromMemory(std::span<const std::byte> data);
```

Comment:
What's the significance of a span of `const T` versus a span of `T`? As in, how does it different from a `const` span of `T`?

Reply:
Span of const T is like pointer to non chanable object. And const span is non changeable span.